File: LeetCode/75.颜色分类.java
class Solution {
    public void sortColors(int[] nums) {
   for (int i = 1; i < nums.length; i++) {
            int j = i;
            int temp =  nums[i];
            while (j > 0 && temp < nums[j - 1]) {
                nums[j] = nums[j - 1];
                j--;
            }
            nums[j] = temp;
        }
    }
}

File: LeetCode/75. 颜色分类.js
/**
 * @param {number[]} nums
 * @return {void} Do not return anything, modify nums in-place instead.
 */
var sortColors = function (nums) {
  for (let i = 1; i < nums.length; i++) {
    let j = i
    let temp = nums[i]
    while (j > 0 && temp < nums[j - 1]) {
      nums[j] = nums[j - 1]
      j--
    }
    nums[j] = temp
  }
}
